src/buffer/out/search.h

```diff
@@ -61,19 +58,21 @@ class Search final
     void _IncrementCoord(COORD& coord) const;
     void _DecrementCoord(COORD& coord) const;
 
-    static COORD s_GetInitialAnchor(const SCREEN_INFORMATION& screenInfo, const Direction dir);
+    COORD s_GetInitialAnchor(const TextBuffer& textBuffer, const Direction dir);
+
     static std::vector<std::vector<wchar_t>> s_CreateNeedleFromString(const std::wstring& wstr);
 
     bool _reachedEnd = false;
     COORD _coordNext = { 0 };
     COORD _coordSelStart = { 0 };
     COORD _coordSelEnd = { 0 };
+    COORD _coordAnchor = { 0 };
 
-    const COORD _coordAnchor;

```

Comment:
Why are we making `_coordAnchor` non-const now?

Reply:
changed